app/models/manageiq/providers/ibm_cloud/inventory.rb

```diff
@@ -6,7 +6,8 @@ class ManageIQ::Providers::IbmCloud::Inventory < ManageIQ::Providers::Inventory
   def self.parsed_manager_name(target)
     case target
     when InventoryRefresh::TargetCollection
-      'VPC::TargetCollection'
+      ems_klass = target.instance_variable_get(:@manager).class
```

Comment:
NOTE we should add `:manager` as an `attr_reader` to `InventoryRefresh::TargetCollection`, PR for this shortly

Reply:
were you going to add manager in the 21272 mentioned above? Or you thinking about adding that at another time?

Reply:
We haven't built the collector instance yet, this is one step above that.  However it might be easier to just pass in `ems` here https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory.rb#L93

Reply:
Okay I went that direction with this, now depends on https://github.com/ManageIQ/manageiq/pull/21273